CHANGELOG.md

```diff
@@ -4,20 +4,29 @@ All notable changes to this project will be documented (for humans) in this file
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [UNRELEASED] - 2018-??-??
-TODO Summary
-### Added
+## [0.5.1] - 2018-06-29
+Bugfix release
 
+### Added
+- Video tutorial / updated slides
+- Helper to set metadata restrictions correctly
+- Usage is now shown when run without arguments
+- New fields to Seqinfo
+  - series_uid
+- Reproin heuristic support for xnat
 ### Changed
-
-### Deprecated
-
+- Dockerfile updated to use `dcm2niix v1.0.20180328`
```

Comment:
while at it -- let's test/update to current (uploaded yesterday to neurodebian) 1.0.20180622?